src/test/bridges/mean/MeanBridgeUnit.t.sol

```diff
@@ -53,6 +57,60 @@ contract MeanBridgeUnitTest is Test {
         assertEq(DAI.allowance(address(bridge), rollupProcessor), type(uint256).max);
     }
 
+    function testRevertsWhenRegisteringWrapperThatIsNotAllowed() public {
+        vm.expectRevert(
+            abi.encodeWithSelector(
+                MeanErrorLib.TokenNotAllowed.selector, 
+                address(DAI)
+            )
+        );
+        
+        address[] memory _tokens = new address[](1);
+        _tokens[0] = address(DAI);
+        _mockIsTokenAllowed(DAI, false);
+        bridge.registerWrappers(_tokens);
+    }
+
+    function testRevertsWhenRegisteringWrapperThatWasAlreadyRegistered() public {        
+        address[] memory _tokens = new address[](1);
+        _tokens[0] = address(DAI);
+        _mockIsTokenAllowed(DAI, true);
+        bridge.registerWrappers(_tokens);
+        vm.expectRevert(
+            abi.encodeWithSelector(
+                MeanErrorLib.TokenAlreadyRegistered.selector, 
+                address(DAI)
+            )
+        );
+        bridge.registerWrappers(_tokens);
+    }
+
+    function testRegisterWrappers() public {
+        address[] memory _tokens = new address[](1);
+        _tokens[0] = address(DAI);
+        
+        vm.expectEmit(true, false, false, false, address(bridge));
+        emit NewWrappersSupported(_tokens);
```

Comment:
Not sure what is happening here. I'm too hardhat head looool.

Reply:
Yeah, I miss hardhat too sometimes. This is basically how we tell Foundry what event to expect